h/models/token.py

```diff
@@ -24,6 +24,9 @@ class Token(Base, mixins.Timestamps):
     #: to, for example, one of the short-lived JWTs that the client uses).
     prefix = u'6879-'
 
+    #: A prefix that identifies a token as a refresh token.
+    refresh_token_prefix = u'7980-'
```

Comment:
If refresh tokens _don't_ have a prefix then it's possible (albeit unlikely) for a randomly generated refresh token to start with the developer or access token prefix, and so look to a human like a different type of token. Anyway, this is unlikely, and people really seem to dislike this refresh token prefix, so I'll remove it.